frontend/src/components/logo.tsx

```diff
@@ -0,0 +1,14 @@
+import logoSymbolBlack from "/logo/ZaneOps-SYMBOL-BLACK.svg";
+import logoSymbolWhite from "/logo/ZaneOps-SYMBOL-WHITE.svg";
+
+export function Logo({ className = "hidden" }: { className: string }) {
+  return (
+    <picture
+      className={`md:${className} w-[100px] h-[100px] flex justify-center items-center`}
```

Comment:
```suggestion
      className={cn("flex justify-center items-center w-[100px] h-[100px]", className )}
```

Don't pass  this, it won't work with tailwindcss, in tailwind we pass the whole className or else tailwind will not be able to discover this class.

Also use the utility `cn` from shadcn, it has a nice feature where classes which affects the same css properties are replaced, so this way if we we have to reuse this component somewhere else with a different width and height, we could just do : `<Logo className="w-8 h-8">` and `cn` would replace the default width classes we passed (`w-[100px h-[100px]`).